frameworks/C/lwan/src/techempower.c

```diff
@@ -53,6 +53,8 @@ static struct db_connection_params {
 static const char hello_world[] = "Hello, World!";
 static const char random_number_query[] =
     "SELECT randomNumber FROM world WHERE id=?";
+static const char cached_random_number_query[] =
+    "SELECT randomNumber FROM world WHERE id=?";
```

Comment:
hey @lpereira I've missed this in the past but for the query tests, we're looking for both `randomNumber` and `id` to be read from the db.

> i. For every request, a single row from a World table must be retrieved from a database table. It is not acceptable to read just the randomNumber.

Reply:
Fair.  Rebased and made a commit making this change.